.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "image": "mcr.microsoft.com/vscode/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/devcontainers/features/go:1": {
+      "version": "1.21"
+    },
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "20"
+    },
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/python:1": {}
+  },
+  "forwardPorts": [
+    8443
+  ],
+  "hostRequirements": {
+    "cpus": 4
```

Comment:
Do we really need 4 cpus to run idp in codespace? The reason I ask, GitHub Free personal account has 120 core compute hours quota per month so if we don't need 4 cpus then we can reduce the compute capacity and buy more time to play with idp within free quota range

Reply:
your right I will remove the cpus requirement